app/src/main/java/org/kiwix/kiwixmobile/zim_manager/local_file_transfer/LocalFileTransferActivity.java

```diff
@@ -0,0 +1,491 @@
+package org.kiwix.kiwixmobile.zim_manager.local_file_transfer;
+
+import android.Manifest;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.content.pm.PackageManager;
+import android.location.LocationManager;
+import android.net.Uri;
+import android.net.wifi.WpsInfo;
+import android.net.wifi.p2p.WifiP2pConfig;
+import android.net.wifi.p2p.WifiP2pDevice;
+import android.net.wifi.p2p.WifiP2pManager;
+import android.os.Build;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.core.content.ContextCompat;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentTransaction;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import kotlin.Unit;
+import kotlin.jvm.functions.Function0;
+import org.kiwix.kiwixmobile.KiwixApplication;
+import org.kiwix.kiwixmobile.R;
+import org.kiwix.kiwixmobile.utils.AlertDialogShower;
+import org.kiwix.kiwixmobile.utils.KiwixDialog;
+import org.kiwix.kiwixmobile.utils.SharedPreferenceUtil;
+
+import java.util.ArrayList;
+
+import javax.inject.Inject;
+
+/**
+ * Created by @Aditya-Sood as a part of GSoC 2019.
```

Comment:
The standard way is to use `@author` followed by the name.

Reply:
The author tag seems to be a very controversial one to use and so I don't want to get into that
I'm just trying to save time for someone who might want to contact me regarding the function of some code in the package, and also to have something tangible for me cherish

Reply:
I generally avoid putting my name in most files
![image](https://user-images.githubusercontent.com/3358282/61520565-6440ba00-aa06-11e9-8970-5d8183015d5a.png)
I have git>annotate bound to a keyboard shortcut so if I need to answer the question "who wrote that?" it is trivial to answer. Comments become obsolete, git history is truth.
I won't require you to remove/update it anyhow but @abdulwd can chime in